api/dispatcher.proto

```diff
@@ -48,6 +49,24 @@ service Dispatcher { // maybe dispatch, al likes this
 	rpc Tasks(TasksRequest) returns (stream TasksMessage) {
 		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-worker" roles: "swarm-manager" };
 	};
+
+	// CreateContainerAttachment allows the node to request the resources
+	// allocation needed for a container attachment on the specific node.
+	rpc CreateContainerAttachment(CreateContainerAttachmentRequest) returns (CreateContainerAttachmentResponse) {
```

Comment:
This is not how the division is made. This is still a declaration of "something needs to attach to a network". That needs to be communicated through the control API.